core-bundle/src/Resources/config/services.yml

```
@@ -746,8 +746,8 @@ services:
             - Contao\BackendUser
             - '@?logger'
 
-    contao.security.data_container.default_dca_voter:
-        class: Contao\CoreBundle\Security\Voter\DataContainer\DefaultDcaVoter
+    contao.security.data_container.default_dc_voter:
+        class: Contao\CoreBundle\Security\Voter\DataContainer\DefaultDcVoter
```

Do we need the `DC` abbreviation here?
```suggestion
        class: Contao\CoreBundle\Security\Voter\DataContainer\DefaultDataContainerVoter
```

---

Done